### DISCUSSION TITLE
Day to day Workflow using Firefly III

### DISCUSSION THREAD

**Discussion by 0xSh1v4** [firefly-iii/firefly-iii#5518] [Firefly III]
Hey Guys, I started using firefly since last week. I have added bills, accounts transactions. how do you guys typically use it as a workflow for your bills when using the credit cards? Is there any way to automate this?I saw the telegram bot where you sent a message and it automatically creates a transaction. unfortunately I couldn't find a docker-image support for arm64 based. Is there any other way instead of manually adding all the bills from credit card? how do we track it? Please pour your suggestions and how you guys use it?
Thanks

**Comment by JC5** [comment]
That's a pretty big question.

You can use the data importer or the telegram bot like you suggest as well. I do it by hand myself, my bank has no automation for my credit card.
